doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,146 @@
 
 <!-- /TOC -->
 
 ## Prerequisites
 
-- Install VS 2019 (Community or higher) and make sure you have the [latest updates](https://www.visualstudio.com/).
+- Install VS 2020 (Community or higher) and make sure you have the [latest updates](https://www.visualstudio.com/).
   - Need at least .NET Framework 4.6.1 and 4.7 development tools
-- Install the **.NET Core cross-platform development** workloads in VisualStudio
-- Install **.NET Core 5.0.301 SDK** for your specific platform. (or a higher version within the 5.0.*** band)  (https://dotnet.microsoft.com/download/dotnet-core/5.0)
+  - Install the **.NET Core cross-platform development** workloads in VisualStudio
+- Install **.NET 6.0 SDK** for your specific platform. (or a higher version)
+- Install **.NET core 3.1** for your specific platform.
 - Install the latest version of [git](https://git-scm.com/downloads)
-- Install [PowerShell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell), version 6 or higher.
+- Install [PowerShell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell), version 7 or higher.
 - Install [NodeJS](https://nodejs.org/) (14.x.x).
 
 ## Setup your repo
 
 - Fork and clone an [azure-sdk-for-net](https://github.com/Azure/azure-sdk-for-net) repo. Follow the instructions in the [.NET Contributing.md](https://github.com/Azure/azure-sdk-for-net/issues/12903) to fork and clone the `azure-sdk-for-net` repo.
 - Create a branch to work in. 
 
-## Getting started
+## Getting started - shippable package
 
-For this tutorial, we'll create a getting started project in a branch of your fork of `azure-sdk-for-net` repo. To create the project, we'll copy the [Azure.Template.Generated](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/template-dpg) project for .NET.
-
-The `Azure.Template.Generated` project is a great place to get started with Azure SDK .NET library development in general, as it contains a number of common patterns we use for tests, samples, documentation, generating API listings. It also has generated code demonstrating an example Data Plane Generated Client.
-
-**Learn more**: see other features listed in the [Azure.Template.Generated README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-dpg/Azure.Template.Generated/README.md).
-
-### Create the project
-
-#### 1. Copy the Azure.Template.Generated directory
-
-Copy the `sdk\template-dpg` directory to an `sdk\<your-service-name>` directory at the same level. For example, from the root of your cloned repo:
+For this guide, we'll create a getting started project in a branch of your fork of `azure-sdk-for-net` repo. We will use dotnet project template [Azure.ServiceTemplate.Template](https://github.com/Azure/azure-sdk-for-net/) to automatically create the project.
 
+You can run `eng\automation\Invoke-DataPlaneGenerateShippablePackage.ps1` to generate the first-version shippable SDK client library package directly as following
```

Comment:
Per comment above, let's call this "starting package" rather than "shippable package".  Let's also define what we mean by that in this section to make it clear to readers, and per @lmazuel, let's make sure we include that it is a package that successfully passes CI in the definition.  Thanks!